vnctl/lib/vnctl/webapi.rb

```diff
@@ -2,7 +2,44 @@
 
 module Vnctl
   class WebApi
-    include HTTParty
-    base_uri "#{Vnctl.conf.webapi_uri}:#{Vnctl.conf.webapi_port}"
+    def initialize
+      @conf = Vnctl.conf
+
+      @output_format = @conf.output_format
+
+      @webapi_full_uri = "%s://%s:%s/api/%s" % [@conf.webapi_protocol,
+                                                @conf.webapi_uri,
+                                                @conf.webapi_port,
+                                                @conf.webapi_version]
+    end
+
+    def send_request(verb, suffix, params = nil)
```

Comment:
> [TooManyStatements]

They're all statements completing the single simple job of making an HTTP request and returning the response body.

> [FeatureEnvy]

`uri` is a local variable only used in this particular requests. These references are fine.